### PR TITLE
Java Solution for Cellular Network (Implementation 1, Binary Search)

### PR DESCRIPTION
**Pull request by brad182**

### CODE REVIEW

File: solutions/cf-702C.mdx [hunk 1]
```diff
@@ -59,9 +59,118 @@ int main() {
...
+	
+<JavaSection>
+	
+```java
+import java.io.*;
+import java.util.*;
+
+public class CellularNetwork {
+
+	static class Kattio extends PrintWriter {
```
Comment (maggie-j-liu) [inline solutions/cf-702C.mdx:73]: remove this

**Review by maggie-j-liu — APPROVED** [review]

**Review by ryanchou-dev — APPROVED** [review]
👍

**Review by ar492 — APPROVED** [review]
i trust code works

File: solutions/cf-702C.mdx [hunk 2]
```diff
@@ -2,7 +2,7 @@
 id: cf-702C
 source: CF
 title: Cellular Network
-author: Nathan Wang, Benjamin Qi, Maggie Liu
+author: Nathan Wang, Benjamin Qi, Maggie Liu, Brad Ma
 ---
 
 [Official Editorial](https://codeforces.com/blog/entry/46324?locale=en)
```
Comment (maggie-j-liu) [inline solutions/cf-702C.mdx:8]: move the solution 1 header outside the language section

File: solutions/cf-702C.mdx [hunk 3]
```diff
@@ -59,9 +59,86 @@ int main() {
...
+
+			int minRForThisCity = TWO_BILLION;
+			// make sure that an index actually exists,
+			// if not, it will return the length of the
+			// array which is equal to m
+			if (towerRight < m) {
+				minRForThisCity = Math.min(minRForThisCity, towers[towerRight] - cities[i]);
+			}
+			if (towerLeft >= 0) {
+				minRForThisCity = Math.min(minRForThisCity, cities[i] - towers[towerLeft]);
```
Comment (maggie-j-liu) [inline solutions/cf-702C.mdx:119]: i think these lines are too long

Comment (brad182) [reply solutions/cf-702C.mdx:119]: but it barely exceeds on only a few lines so i think it should be fine

Comment (maggie-j-liu) [reply solutions/cf-702C.mdx:119]: the lines wrap on my screen but ok

**Pull request merged**